index.js

```diff
@@ -16,10 +16,12 @@ function simpleGet (opts, cb) {
     cb(new Error('too many redirects'))
     return
   }
-  if (!opts.maxRedirects) opts.maxRedirects = 10
+
+  opts.followRedirects = typeof opts.followRedirects === 'boolean' ? opts.followRedirects : true
```

Comment:
We don't need a new option for this. We can just use the existing `maxRedirects` option. If `maxRedirects === 0` then no redirects should be followed.